hypertrace-ingester/src/test/java/org/hypertrace/ingester/HypertraceIngesterTest.java

```diff
@@ -0,0 +1,115 @@
+package org.hypertrace.ingester;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.google.protobuf.ByteString;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.jaegertracing.api_v2.JaegerSpanInternalModel.Span;
+import java.io.File;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.TestInputTopic;
+import org.apache.kafka.streams.TestOutputTopic;
+import org.apache.kafka.streams.TopologyTestDriver;
+import org.hypertrace.core.kafkastreams.framework.serdes.AvroSerde;
+import org.hypertrace.core.serviceframework.config.ConfigClientFactory;
+import org.hypertrace.core.spannormalizer.jaeger.JaegerSpanSerde;
+import org.hypertrace.traceenricher.trace.enricher.StructuredTraceEnricherConstants;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.junitpioneer.jupiter.SetEnvironmentVariable;
+
+/**
+ * Topology Test for {@link HypertraceIngester}
+ */
+public class HypertraceIngesterTest {
```

Comment:
It is really great we finally have such test. A couple of things I'd like to mention:
1. I would suggest we describe what the test does and verifies in the name e.g. TestSpanEventIsProcessedByThePipeline
2. I see there are many things to bootstrap here. I would suggest we bootstrap them in a separated static methods so we remove the distraction from the test and make it more readable.
3. I'd really encourage you to add a basic span with a set of attributes and display the expected result at the end of the pipeline. Nothing fancy but doing minimal checks can get us a good idea of what is happening. In general I would cover the main use cases here (so far the main use case is that we put a span and get a spanEvent at the end).

Reply:
Thanks JC for the review. 
2 is done. 
For 1 & 3, I would like share my idea for the test. The purpose of this test was to ensure basic sanity for ingester(ie. the topologies are connected, serde config...). I believe test for any business logic should remain in the respective service(module), that is why I have named it `testIngestionPacketFlow`. Obvious question is why only test SpanView generation? because that's the most simple use case and serves the purpose of the test. Having test named `TestSpanEventIsProcessedByThePipeline` would imply we plan to add test for other events as well?

Reply:
I think the idea is good, my main concern is that I think we should do at least really basic comparisons (e.g. how we make sure the output record isn't an empty object). I don't think we should look at complex stuff like span normalization of attributes but instead traceID and spanID should be there for sure (timestamp and duration too but there is some changes in units that we don't want to tackle as of now, see https://github.com/hypertrace/hypertrace/issues/93
)

Reply:
Yeah, basic check is required. Added it.